Formula/twine-pypi.rb

```diff
@@ -0,0 +1,45 @@
+class TwinePypi < Formula
+  include Language::Python::Virtualenv
+
+  desc "Utilities for interacting with PyPI"
+  homepage "https://pypi.python.org"
+  url "https://github.com/pypa/twine/archive/1.8.1.tar.gz"
+  sha256 "918d7ebab52280bac573af9a6c80622202f0cdeab14cb3360f9ffb9848be946d"
+  head "https://github.com/pypa/twine.git"
+
+  depends_on :python
```

Comment:
Actually, we have far more instances of `depends_on :python if MacOS.version <= :snow_leopard` (i.e. without `:recommended`).